src/advanced_fees_management.tf

```diff
@@ -0,0 +1,178 @@
+# region
+resource "azurerm_resource_group" "advanced_fees_management_rg" {
+  name     = format("%s-afm-rg", local.project)
+  location = var.location
+
+  tags = var.tags
+}
+
+# local variables
+locals {
+  advanced_fees_management_cors_configuration = {
+    origins = ["*"]
+    methods = ["*"]
+  },
+  advanced_fees_management_cosmosdb_enable_serverless = contains(var.advanced_fees_management_cosmosdb_extra_capabilities, "EnableServerless")
+}
+
+# subnet
+module "advanced_fees_management_snet" {
+  count                                          = var.advanced_fees_management_enabled && var.cidr_subnet_advanced_fees_management != null ? 1 : 0
```

Comment:
I'd be more explicit I'd test env variable directly. It's a PoC

Reply:
done